giskard/ml_worker/websocket/listener.py

```diff
@@ -86,6 +86,9 @@ def handle_result(future: Future):
 
         try:
             info: websocket.WorkerReply = future.result()
+        except CancelledError:
+            info: websocket.WorkerReply = websocket.Empty()
```

Comment:
According to @kevinmessiaen , the empty response could cause problem on Java side. Could it be better to merge https://github.com/Giskard-AI/giskard-hub/pull/134 first?

Reply:
Sure about the PR.
It was looking fine from what I saw when testing. What should I send back ?

Reply:
I think it's fine with @kevinmessiaen 's PR.
At the moment the empty response is just ignored. I guess it's OK with the push feature because there is no transaction.
Maybe we can add a comment here to remind people to handle the empty response when adding a transactional action.

Reply:
@Inokinoki Yes it's okay to return an `Empty` message in that case, the issue was if an error arose it wasn't intercepted because we didn't read the response DTO returned by the worker in the backend.

It caused issue as we didn't rolled back transactions when error wasn't intercepted properly. But there since we cancel  an action it's not an issue (and the issue will be fixed when the PR is merged anyway)